src/src/rx_main.cpp

```diff
@@ -825,7 +825,8 @@ void loop()
         RFmodeLastCycled = millis();
     }
 
-    if ((connectionState == connected) && ((millis() > (LastValidPacket + ExpressLRS_currAirRate_RFperfParams->RFmodeCycleInterval)) || ((millis() > (LastSyncPacket + 11000)) && uplinkLQ < 10))) // check if we lost conn.
+    uint32_t localLastValidPacket = LastValidPacket; // Required to prevent race condition due to LastValidPacket getting updated from ISR
```

Comment:
We need to debug why this is needed before merge. 
There has to be something else which is causing failures and I am a bit worried that this does not fix those permanently...